src/Stripe.net.Tests/Stripe.net.Tests.csproj

```diff
@@ -31,22 +52,4 @@
     <EmbeddedResource Include="infrastructure\test_data\subscription.json" />
   </ItemGroup>
 
```

Comment:
technically it was half right - it needs to pull in the assemblies (dotnet restore) :)